afni_proc_hw/exp_1/proc.sub_03.test

```diff
@@ -0,0 +1,481 @@
+1d_tool.py -infile dfile_rall.1D -set_nruns 1                                \
+           -demean -write motion_demean.1D
+
+# compute motion parameter derivatives (for use in regression)
+1d_tool.py -infile dfile_rall.1D -set_nruns 1                                \
+           -derivative -demean -write motion_deriv.1D
+
+# create censor file motion_${subj}_censor.1D, for censoring motion 
+1d_tool.py -infile dfile_rall.1D -set_nruns 1                                \
+    -show_censor_count -censor_prev_TR                                       \
+    -censor_motion 0.5 motion_${subj}
+
+# note TRs that were not censored
+set ktrs = `1d_tool.py -infile motion_${subj}_censor.1D                      \
+                       -show_trs_uncensored encoded`
+
+# ------------------------------
+# run the regression analysis
+3dDeconvolve -input pb04.$subj.r*.scale+tlrc.HEAD                            \
+    -censor motion_${subj}_censor.1D                                         \
+    -polort 4                                                                \
+    -num_stimts 17                                                           \
+    -stim_times 1 stimuli/correct_task.1D 'BLOCK(15)'                        \
+    -stim_label 1 correct_task                                               \
+    -stim_times 2 stimuli/incorrect_task.1D 'BLOCK(15)'                      \
+    -stim_label 2 incorrect_task                                             \
+    -stim_times 3 stimuli/no_response_control.1D 'BLOCK(15)'                 \
+    -stim_label 3 no_response_control                                        \
+    -stim_times 4 stimuli/no_response_task.1D 'BLOCK(15)'                    \
+    -stim_label 4 no_response_task                                           \
+    -stim_times 5 stimuli/response_control.1D 'BLOCK(15)'                    \
+    -stim_label 5 response_control                                           \
+    -stim_file 6 motion_demean.1D'[0]' -stim_base 6 -stim_label 6 roll_01    \
+    -stim_file 7 motion_demean.1D'[1]' -stim_base 7 -stim_label 7 pitch_01   \
+    -stim_file 8 motion_demean.1D'[2]' -stim_base 8 -stim_label 8 yaw_01     \
+    -stim_file 9 motion_demean.1D'[3]' -stim_base 9 -stim_label 9 dS_01      \
+    -stim_file 10 motion_demean.1D'[4]' -stim_base 10 -stim_label 10 dL_01   \
+    -stim_file 11 motion_demean.1D'[5]' -stim_base 11 -stim_label 11 dP_01   \
+    -stim_file 12 motion_deriv.1D'[0]' -stim_base 12 -stim_label 12 roll_02  \
+    -stim_file 13 motion_deriv.1D'[1]' -stim_base 13 -stim_label 13 pitch_02 \
+    -stim_file 14 motion_deriv.1D'[2]' -stim_base 14 -stim_label 14 yaw_02   \
+    -stim_file 15 motion_deriv.1D'[3]' -stim_base 15 -stim_label 15 dS_02    \
+    -stim_file 16 motion_deriv.1D'[4]' -stim_base 16 -stim_label 16 dL_02    \
+    -stim_file 17 motion_deriv.1D'[5]' -stim_base 17 -stim_label 17 dP_02    \
+    -gltsym 'SYM: response_control -0.5*correct_task -0.5*correct_task'      \
```

Comment:
We want the task > control contrast, so you need to flip the signs here:
`-gltsym 'SYM: -response_control 0.5*correct_task 0.5*correct_task'`

These contrast give the same results with a sign flip, but since we do 1-sided thresholding of positive clusters later, this will give you wrong result.